src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar page__calendar calendar--start-day--friday calendar--month-length--28">
+      <div class="calendar__date calendar__date--cell--1"></div>
+      <div class="calendar__date calendar__date--cell--2"></div>
+      <div class="calendar__date calendar__date--cell--3"></div>
+      <div class="calendar__date calendar__date--cell--4"></div>
+      <div class="calendar__date calendar__date--cell--5"></div>
+      <div class="calendar__date calendar__date--cell--6"></div>
+      <div class="calendar__date calendar__date--cell--7"></div>
+      <div class="calendar__date calendar__date--cell--8"></div>
+      <div class="calendar__date calendar__date--cell--9"></div>
+      <div class="calendar__date calendar__date--cell--10"></div>
+      <div class="calendar__date calendar__date--cell--11"></div>
+      <div class="calendar__date calendar__date--cell--12"></div>
+      <div class="calendar__date calendar__date--cell--13"></div>
+      <div class="calendar__date calendar__date--cell--14"></div>
+      <div class="calendar__date calendar__date--cell--15"></div>
+      <div class="calendar__date calendar__date--cell--16"></div>
+      <div class="calendar__date calendar__date--cell--17"></div>
+      <div class="calendar__date calendar__date--cell--18"></div>
+      <div class="calendar__date calendar__date--cell--19"></div>
+      <div class="calendar__date calendar__date--cell--20"></div>
+      <div class="calendar__date calendar__date--cell--21"></div>
+      <div class="calendar__date calendar__date--cell--22"></div>
```

Comment:
i don't think you should hardcode the cell numbers although there is nothing against it in the rules, maybe you can use for loop or nth child or both.

Reply:
agreed. nth-child and a loop is the way to go

Reply:
Done